src/component/common/FormTemplate/FormTemplate.styles.ts

```diff
@@ -1,5 +1,7 @@
 import { makeStyles } from '@material-ui/core/styles';
 
+export const formTemplateSidebarWidth = '27.5rem';
```

Comment:
Should this make it into the theme? There's a discussion regarding how to use the theme properly that I'd like to have at some point.

Reply:
Not sure, quite hacky at the moment. Only used by the form steps container, to center it above the main form column.